readthedocs/api/v3/serializers.py

```diff
@@ -59,10 +60,33 @@ class Meta:
         fields = []
 
 
+# TODO: decide whether or not include a `_links` field on the object
```

Comment:
Notifications probably don't need a detail view, but it would help to have a dismiss URL. Right now, we do this at the template load time:

https://github.com/readthedocs/ext-theme/blob/4bfe050e5f4db2855490a47e0524685c8bad33a2/readthedocsext/theme/templates/includes/utils/messages.html#L16

We won't be able to do this with the new notification system. It's best that the front end never have to recreate URL creation logic, so I think we do want some links returned on the notification API response.

Reply:
Yes, ideally, all these URLs should be returned in the API response, as we are doing with other resources with the `_links` attribute: https://docs.readthedocs.io/en/stable/api/v3.html#project-details

Then, the frontend can just rely on them being always correct 👍🏼

Reply:
I will make these changes in a following PR when we start using APIv3 from the front-end.